reference/test/solver/batch_bicgstab_kernels.cpp

```diff
@@ -136,7 +141,7 @@ TYPED_TEST(BatchBicgstab, CanSolveDenseSystem)
     using real_type = gko::remove_complex<value_type>;
     using Solver = typename TestFixture::solver_type;
     using Mtx = typename TestFixture::Mtx;
-    const real_type tol = 1e-5;
+    const real_type tol = 1e-4;
```

Comment:
Maybe loosen the tolerance only for half ? I think it would be better to leave the tighter tolerance for the other types.

Reply:
I think 1e-5 is quite loose for double, so 1e-5 and 1e-4 does not make big difference there.
I can change them only in half precision.
Side note. this condition also means that it is more challenging to half than double/single precision on the machine eps perspective.